### PR TITLE
DOCSP-40527 100.9.5 Changelog

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

- Adds `100.9.5` release notes.
- Fixes all error occurrences of `1.9` - `100.9.0` version references for all versions. DBtools versions start at version `100.0.0`.
- Renamed release note files with correct versions i.e. `db-tools-1.0-changelogs.txt` -> `db-tools-100.0.0-changelogs.txt`. 

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/database-tools/DOCSP-40527/release-notes/database-tools-changelog/

## JIRA

https://jira.mongodb.org/browse/DOCSP-40527

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=667441b3ebce19f3148d9395

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/changelogs/dbtools-changelogs.rst [hunk 1]
```diff
@@ -10,38 +10,42 @@ releases.
     - Release Dates
     - MongoDB Server Support
 
-  * - :ref:`Database Tools 1.8.0 <dbtools-1.8-changelog>`
+  * - :ref:`Database Tools 100.9.0 - 100.9.5 <dbtools-100.9.0-changelog>`
```
Comment (autarch) [inline source/includes/changelogs/dbtools-changelogs.rst:13]: Does this end up linking to the 100.9.5 notes? I don't really understand the details of how this is structured.

**Review by autarch — APPROVED** [review]
LGTM % one question.

File: source/includes/changelogs/dbtools-changelogs.rst (hunk 1 above)
Comment (ianf-mongodb) [reply source/includes/changelogs/dbtools-changelogs.rst:13]: Yeah, it is a little confusing for me as well, but if you click the link it loads the `100.9.0` - `100.9.5` content.
A better way to test is:

1. Click the `100.7.0` release notes in the side nav
2. Click the `Database Tools 100.9.0 - 100.9.5` link at the top of the page, that should bring back the `100.9.0` RN's. 

@autarch

**Review by mdb-ashley — APPROVED** [review]
LGTM, thanks Ian!

**Pull request merged**